src/main/groovy/com/github/benmanes/gradle/versions/updates/Resolver.groovy

```diff
@@ -117,6 +118,15 @@ class Resolver {
       createQueryDependency(dependency, revision)
     }
 
+    // Common use case for dependency constraints is a java-platform BOM project.
+    try {
+      configuration.dependencyConstraints.each { dependency ->
+        latest.add(createQueryDependency(dependency, revision))
+      }
+    } catch (MissingPropertyException e) {
```

Comment:
In other cases we've guarded using `metaClass.respondsTo("...")`. Which do you consider preferable?